src/components/AddressBook/AddressBookThirdParty/AddressBookThirdParty.tsx

```diff
@@ -8,27 +8,34 @@ interface AddressBookThirdPartyProps {
   thirdPartyPageResults: AddressBookThirdPartyResultsProps[];
   isSearchingThirdParty: boolean;
   network: string;
+  hasEntityLookupPath: boolean;
 }
 
 export const AddressBookThirdParty: FunctionComponent<AddressBookThirdPartyProps> = ({
   onAddressSelect,
   thirdPartyPageResults,
   isSearchingThirdParty,
   network,
+  hasEntityLookupPath,
 }) => {
   return (
     <table className="table">
       <thead className="table-thead">
         <tr>
-          <th>Name</th>
+          <th className="w-20">Name</th>
           <td>Address</td>
-          <td>Source</td>
           <td>&nbsp;</td>
+          <td>Source</td>
         </tr>
       </thead>
       <tbody className="table-tbody">
         {isSearchingThirdParty ? (
           <AddressBookTableRowEmpty message="Searching..." />
+        ) : !hasEntityLookupPath ? (
+          <AddressBookTableRowEmpty
+            message="This address book’s endpoint does not have the entityLookup feature, do contact the respective personnal to set it up."
+            textClassName="text-red"
+          />
         ) : isEmpty(thirdPartyPageResults) ? (
           <AddressBookTableRowEmpty message="No address found. Try searching?" />
         ) : (
```

Comment:
maybe can refactor this abit to a switch function or smt. cos seems like alot of ternary operator. quite hard to read and maintain later.

Reply:
ok sure